etc/notebooks/test/test_scala_hello_world.ipynb

```diff
@@ -0,0 +1,67 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "collapsed": false,
+    "urth": {
+     "dashboard": {
+      "layout": {
+       "col": 0,
+       "height": 4,
+       "row": 0,
+       "width": 4
+      }
+     }
+    }
+   },
+   "outputs": [
```

Comment:
Put a markdown cell somewhere in the dashboard. The hello world took so long to appear, I thought my dashboard server was busted. At least the markdown will appear instantly and give some sign of life. (Odd that the spinner bar didn't appear either ...)